receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -54,7 +54,11 @@ var standardMetrics = []string{
 	"system.network.io",
 	"system.network.packets",
 	"system.paging.operations",
-	"system.paging.usage",
+}
+
+var archSpecificMetrics = map[string][]string{
```

Comment:
If I understand correctly, this change is caused by the fact that the Host Metrics receiver's test `TestGatherMetrics_EndToEnd` is tightly coupled to the underlying operating system specifics. In this case, depending on whether the system has a swap partition or not, the test will pass or fail. The runners from Actuated apparently do not have swap partition, and this is why the metric `system.paging.usage` is not created on them, causing the tests to fail.

This change tries to alleviate it by expecting that this metric does not exist on ARM64. This is not strictly correct, because whether this metric is created or not does not depend on the architecture, but on the specific machine the test is run on. For example, the original test passes on my Raspberry Pi (which has a swap partition), but it will fail after this change.

This should probably be fixed by making the test pass independently of whether the underlying system has a swap partition or not.

Reply:
I'm not sure what you want here. The tests already do a lot of interesting things to pass for Windows for example.

Reply:
I mostly wanted to see your response to my messy thoughts 😄 I think the two options we have are:

1. Merge this PR in its current state, with the fix to unit tests that is tailored specifically to our CI and the runners we use
2. Remove the unit test fixes from this PR and only merge the addition of new runners, letting the Host Metrics receiver unit tests to fail on Actuated runners until this is resolved (properly) on the Host Metrics receiver side.

I suppose 1. is fine. As you point out, it doesn't make the Host Metrics receiver unit tests much worse.

Reply:
I really don't know. I think the codeowners of the receiver might need to do some soul searching and create end to end tests at some point in some really controlled environments to make sure everything comes in properly.